modules/pipeline/conf/conf.go

```diff
@@ -112,6 +112,9 @@ type Conf struct {
 	// scheduler executor refresh interval
 	ExecutorRefreshIntervalMinute uint64 `env:"EXECUTOR_REFRESH_INTERVAL_MINUTE" default:"20"`
 	SpecifyImagePullPolicy        string `env:"SPECIFY_IMAGE_PULL_POLICY" default:"IfNotPresent"`
+
+	// k8s namespace
+	EnableSpecifiedK8SNamespace string `env:"ENABLE_SPECIFIED_K8S_NAMESPACE"`
```

Comment:
I think `specified_k8s_namespace` should be a request level config.

Reply:
adjusted, only use `task.Extra.NotPipelineControlledNs` 

NotPipelineControlledNs is request level config